tendrl/ceph_integration/flows/create_pool.py

```diff
@@ -1,6 +1,16 @@
-from tendrl.ceph_integration.flows.flow import Flow
+from tendrl.ceph_integration.manager.crud import Crud
+from tendrl.ceph_integration.manager import utils as manager_utils
+from tendrl.common.flows.base_flow import BaseFlow
 
 
-class CreatePool(Flow):
+class CreatePool(BaseFlow):
     def run(self):
+        self.parameters.update(
```

Comment:
Thats correct. we can import crud module and use the functions as is. Will change the logic accordingly.